Content.Shared/Throwing/LandEvent.cs

```diff
@@ -5,31 +5,13 @@
 namespace Content.Shared.Throwing
 {
     /// <summary>
-    ///     Raised when an entity that was thrown lands.
+    ///     Raised when an entity that was thrown lands. This occurs before they stop moving and is when their tile-friction is reapplied.
     /// </summary>
     [PublicAPI]
-    public sealed class LandEvent : EntityEventArgs
-    {
-        /// <summary>
-        ///     Entity that threw the item.
-        /// </summary>
-        public IEntity? User { get; }
+    public sealed class LandEvent : EntityEventArgs {}
 
-        /// <summary>
-        ///     Item that was thrown.
-        /// </summary>
-        public IEntity Thrown { get; }
-
-        /// <summary>
-        ///     Location where the item landed.
-        /// </summary>
-        public EntityCoordinates LandLocation { get; }
-
-        public LandEvent(IEntity? user, IEntity thrown, EntityCoordinates landLocation)
-        {
-            User = user;
-            Thrown = thrown;
-            LandLocation = landLocation;
-        }
```

Comment:
was this all just removed because nothing used it?

Reply:
At the very least, keep the `User` one imo

Reply:
Okey dokey